### PR TITLE
chore: rename default branch to main

### PR DESCRIPTION
**Pull request by glennreyes**
This applies changes in the codebase in preparation for updating our default branch to main.

### Important steps to do before merging this:

- [x] Create and push a `main` branch that reflects current `master`
- [x] Switch default branch to `main` in GitHub repository settings (cc @alber70g @Randynamic)
- [ ] Announce team contributors to update local git config at `~/.gitconfig` to add the snippet below (less friction afterwards when creating new repos)
- [x] Delete master branch

### Important steps after merging:

- [x] Update base branches in open PRs to `main` on GitHub

### `.gitconfig`

```
[init]
	defaultbranch = main
```

Closes https://app.asana.com/0/1202676524474548/1204615060102551/f
Ref: https://www.theserverside.com/feature/Why-GitHub-renamed-its-master-branch-to-main

### CODE REVIEW

File: packages/apps/kadena-docs/src/pages/docs/build/guides/building-a-voting-dapp.mdx [hunk 1]
```diff
@@ -699,10 +699,9 @@ interface. The `gas-payer-v1` interface is deployed to all chains on `testnet`
 and `mainnet` so you can directly use it in your contract. We can specify that a
 module implements an interface using the `(implements INTERFACE)` construct. :::
 
-:::info  
-Pact interfaces are similar to Java's interfaces, Scala's traits, Haskell's
-typeclasses or Solidity's interfaces. If you're not familiar with this concept
-you can
+:::info Pact interfaces are similar to Java's interfaces, Scala's traits,
+Haskell's typeclasses or Solidity's interfaces. If you're not familiar with this
```
Comment (glennreyes) [inline packages/apps/kadena-docs/src/pages/docs/build/guides/building-a-voting-dapp.mdx:703]:
```suggestion
:::info
 Pact interfaces are similar to Java's interfaces, Scala's traits,
Haskell's typeclasses or Solidity's interfaces. If you're not familiar with this
```

File: packages/apps/kadena-docs/src/pages/docs/build/guides/building-a-voting-dapp.mdx [hunk 2]
```diff
@@ -699,10 +699,10 @@ interface. The `gas-payer-v1` interface is deployed to all chains on `testnet`
 and `mainnet` so you can directly use it in your contract. We can specify that a
 module implements an interface using the `(implements INTERFACE)` construct. :::
 
-:::info  
-Pact interfaces are similar to Java's interfaces, Scala's traits, Haskell's
-typeclasses or Solidity's interfaces. If you're not familiar with this concept
-you can
+:::info
+ Pact interfaces are similar to Java's interfaces, Scala's traits,
```
Comment (glennreyes) [inline packages/apps/kadena-docs/src/pages/docs/build/guides/building-a-voting-dapp.mdx:703]:
```suggestion
:::info
Pact interfaces are similar to Java's interfaces, Scala's traits,
```

**Review by webpro — APPROVED** [review]
LGTM 🤣 (sorry j/k)

**Review by Randynamic — APPROVED** [review]

**Comment by Randynamic** [comment]
I think we can just rename master to main and merge this PR after.
We don't need to keep a shadow branch as github redirects master -> main if we rename

**Comment by glennreyes** [comment]
> I think we can just rename master to main and merge this PR after. We don't need to keep a shadow branch as github redirects master -> main if we rename

Indeed! References are redirecting correctly to the main branch. However, this PR includes updates in the workflows that replaces explicitly specified `master` to `main` in the config files and these won't be covered by GH.

**Pull request merged**